coalib/bears/Bear.py

```diff
@@ -305,6 +316,33 @@ def log_message(self, log_message, timestamp=None, **kwargs):
     def run(self, *args, dependency_results=None, **kwargs):
         raise NotImplementedError
 
+    def _dump_bear_profiled_file(self, profiler):
+        filename = '{}_{}.prof'.format(self.section.name, self.name)
+        if self.profile is True:
+            profiler.dump_stats(filename)
+        else:
+            if not (os.path.exists(self.profile) and
+                    os.path.isdir(self.profile)):
+                os.mkdir(os.path.join(self.profile))
```

Comment:
what if a file exists with that name? won't that raise an exception?

Reply:
If I understand full then then `FileExitsError` will raised, when user give `filepath`, and IMO that error should have to be happen here.